articles/libraries/lock/v10/migration-guide.md

```diff
@@ -4,39 +4,74 @@ Heads up! This document is using the latest version of Lock (version 10). See ch
 
 # Lock 9 to Lock 10 Migration Guide
 
-The following instructions assume you are migrating from Lock 9 to the latest Lock 10. If you are upgrading from a preview release, please refer to the [preview changes](#preview-changes). If you came here looking for the migration guide from the "Auth0Widget" to Lock v9, here's the [legacy-migration-guide](/libraries/lock/v9/migration-guide). Otherwise, read on!
+The following instructions assume you are migrating from **Lock 9** to the latest **Lock 10**. If you are upgrading from a preview release of Lock 10, please refer to the [preview changes](#preview-changes). Otherwise, read on!
 
-- The `show` method no longer takes any arguments. You pass the options to the constructor and you listen for an `authenticated` event instead of providing a callback. The examples above show how to listen for this event with the `on` method.
-- The `authenticated` event listener has a single argument, an `authResult` object. This object contains the following properties: `idToken`, `accessToken`, `state`, `refreshToken` and `idTokenPayload`. Most of them correspond to the arguments passed to the `show` method's callback.
-- The profile is no longer fetched automatically after a successful login, you need to call `lock.getProfile`.
-- Lock now uses Redirect Mode by default. To use Popup Mode, you must enable this explicitly with the `auth: { redirect: false }` option.
+## General Changes and Additions
+
+### User Profiles
+- The profile is no longer fetched automatically after a successful login, you need to call [lock.getProfile](/libraries/lock/v10/api#getprofile-token-callback-).
+
+### Redirect Mode vs Popup Mode
+- Lock now uses Redirect Mode by default. To use [Popup Mode](/libraries/lock/v10/popup-mode), you must enable this explicitly with the [redirect](/libraries/lock/v10/customization#redirect-boolean-) option `auth: { redirect: false }`.
 - You no longer need to call `parseHash` when implementing Redirect Mode. The data returned by that method is provided to the `authenticated` event listener.
+
+### Customizing Options
+- The `show` method no longer takes any arguments. You pass the options to the constructor and you listen for an `authenticated` event instead of providing a callback. You can listen for this event with the [on](/libraries/lock/v10/api#on-event-callback-) method.
+
+### Authenticated Event
+- The `authenticated` event listener has a single argument, an `authResult` object. This object contains the following properties: `idToken`, `accessToken`, `state`, `refreshToken` and `idTokenPayload`. Most of them correspond to the arguments passed to the `show` method's callback.
+
+### Internationalization
 - Not all languages supported by Lock v9 are supported by Lock v10.

```

Comment:
@jeffreylees Shouldn't we list the languages not supported or redirect to a doc that has this info?